lib/HotModuleReplacementPlugin.js

```diff
@@ -564,7 +565,7 @@ class HotModuleReplacementPlugin {
 													newRuntime
 											  );
 										if (hash !== oldHash) {
-											if (module.type === "runtime") {
+											if (module.type === WEBPACK_MODULE_TYPE_RUNTIME) {
```

Comment:
There are still instances of raw "runtime" strings however these are being used in a different way in the template classes and didn't feel appropriate to replace with this, which is specifically reserved for `module.type` references.